manifests/service.pp

```diff
@@ -65,5 +77,40 @@
       }
     }
   }
+}
+define filebeat::service::add {
```

Comment:
Can we split the defined type in to its [own file](https://puppet.com/docs/puppet/7.1/style_guide.html#separate-files).

We might want to think about titling this one a little different too since we're not creating a new "add". I don't have another idea yet though.